### PR TITLE
Add requires.io filters to requirements.txt

### PR DESCRIPTION
**Pull request by hayfield**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,33 +2,32 @@
...
-rq==0.8.1
-Unidecode==0.04.21
-requests==2.18.3
-Flask-RQ==0.2
-flask-heroku==0.1.9
-Flask-And-Redis==0.7
-Flask-Markdown==0.3
-xmltodict==0.11.0
-gevent>=0.13.8
+Flask==0.12.2  # rq.filter: <1.0
```
Comment (andylolz) [inline requirements.txt]: For consistency, I think this one should be:

```
Flask==0.12.2  # rq.filter: <0.13
```

Comment (hayfield) [reply requirements.txt]: So it should!

Comment (hayfield) [reply requirements.txt]: 4102609ad13cc363b15c8f18aeb47a3fdf3213a1

**Comment by dalepotter** [comment]
Tests are failing :-(

`ImportError: No module named flask.ext.sqlalchemy._MapperSignalEvents`

**Comment by hayfield** [comment]
Googling of the error message indicates that it means the tests are not running from within a virtualenv.

**Comment by andylolz** [comment]
This is a branch from `requires-io-master`, which is also failing tests (presumably because requires.io is updating packages on that branch without these sorts of filters!)

I think you’d be better off branching from `master` (where the build is passing) and adding these sorts of filters there.

**Comment by andylolz** [comment]
> Googling of the error message indicates that it means the tests are not running from within a virtualenv.

FWIW if I create a new virtualenv locally, install the dev requirements with pip, then run tests, I hit the same error message.

**Comment by hayfield** [comment]
Will create a new branch from master for these changes, then close this and #266

**Comment by hayfield** [comment]
(I'll leave the `live` -> `master` -> `dev` change to be a separate task since it has the potential to break things and there's no working Dev Datastore at the moment)

**Comment by hayfield** [comment]
#288 does this in a hopefully-working manner (by removing months of not-OK changes).

**Pull request closed without merging**